GNUmakefile

```diff
@@ -437,7 +437,7 @@ clang:
 #
 # Run the "make cocci" target to just output what would be done, or "make cocci-inplace"
 # to apply the changes to the source tree.
-COCCI_FILES ?= maint/coccinelle/*.cocci
+COCCI_FILES ?= devel/coccinelle/*.cocci
```

Comment:
I would move all the cocci stuff to devel/Makefile.am

Reply:
And then we run `make -C devel cocci`?

Reply:
exactly